### PR TITLE
Update Pure Storage URL

### PR DESCRIPTION
**Pull request by sdodsley**
Update URL to point to new GitHub repo for driver
```release-note
NONE
```

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -72,7 +72,11 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [Portworx](https://github.com/libopenstorage/openstorage/tree/master/csi) | `pxd.openstorage.org` | v0.3, v1.1 | A Container Storage Interface (CSI) Driver for [Portworx](https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/csi/) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion
+<<<<<<< HEAD
 [Pure Storage CSI](https://github.com/purestorage/helm-charts)| `pure-csi` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for Pure Storage's [Pure Service Orchestrator](https://purestorage.com/containers) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Cloning, Raw Block, Topology, Expansion
+=======
+[Pure Storage CSI](https://github.com/purestorage/pure-pso)| `pure-csi` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for Pure Storage's [Pure Service Orchestrator](https://purestorage.com/containers) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Cloning, Raw Block, Topology, Resize
+>>>>>>> Update Pure Storage URL
```
Comment (xing-yang) [inline book/src/drivers.md:79]: Can you remove these (line 75, 77, and 79)?  Looks like that's from a merge conflict.

Comment (xing-yang) [reply book/src/drivers.md:79]: Actually line 76 needs to be removed as well.  Otherwise you'll have both old and new entries there.

Comment (sdodsley) [reply book/src/drivers.md]: Resolved

File: book/src/drivers.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [Open-E](https://github.com/open-e/JovianDSS-KubernetesCSI) | `com.open-e.joviandss.csi` | v1.0 | A Container Storage Interface (CSI) Driver for Open-E JovianDSS Storage  | Persistent | Read/Write Single Pod | Yes | Snapshot, Cloning 
 [Portworx](https://github.com/libopenstorage/openstorage/tree/master/csi) | `pxd.openstorage.org` | v0.3, v1.1 | A Container Storage Interface (CSI) Driver for [Portworx](https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/csi/) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion
-[Pure Storage CSI](https://github.com/purestorage/helm-charts)| `pure-csi` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for Pure Storage's [Pure Service Orchestrator](https://purestorage.com/containers) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Cloning, Raw Block, Topology, Expansion
+[Pure Storage CSI](https://github.com/purestorage/pure-pso)| `pure-csi` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for Pure Storage's [Pure Service Orchestrator](https://purestorage.com/containers) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Cloning, Raw Block, Topology, Resize
```
Comment (xing-yang) [inline book/src/drivers.md:75]: Is "https://github.com/purestorage/pure-pso" the right link?  It gave me a "404" error.

Comment (xing-yang) [reply book/src/drivers.md:75]: "Expansion" is used for all other drivers in this doc.  Can you keep this as "Expansion"?

Comment (sdodsley) [reply book/src/drivers.md]: @xing-yang fixed

**Pull request merged**